packages/design/src/ButtonIcon/ButtonIcon.jsx

```diff
@@ -50,6 +50,7 @@ const fromProps = props => {
   return {
     '&:disabled': {
       color: theme.colors.action.disabled,
+      pointerEvents: 'none',
```

Comment:
We might be better off solving it this way:

```js
    '&:disabled': {
      color: theme.colors.action.disabled,
    },
    '&:hover:enabled, &:focus:enabled': {
      background: theme.colors.action.hover,
    },
```

I just tested this and it still works, I suppose you wanted to remove the hover effect when the file transfer box is shown?

`pointerEvents: none` strikes me as a very substantial change and I'm worried that there are some places in the code which depend on `ButtonIcon` being a target of pointer events, even when disabled.

Reply:
It works great, thanks!
I also added `cursor: default` when disabled.